### ISSUE TITLE
Set/push of another json with a mapper would not change node names

### ISSUE THREAD

**Issue by sbcgua** [sbcgua/ajson#102]
didn't test but I don't think there is a code for this, so a potential bug to test and fix

```abap
li_json_w_mapper->set( iv_path = 'a' iv_val = '1' ). " e.g. a -> b
li_json->set( iv_path = '/xyz' iv_val = li_json_w_mapper ).
```

**Comment by sbcgua** [comment]
Would be implicitly solved by #112

**Comment by sbcgua** [comment]
Closing, the issue became irrelevant after mapper approach change in #117
